guides/v2.3/rest/modules/inventory/manage-source-selection.md

```diff
@@ -0,0 +1,140 @@
+---
+group: rest-api
+title: Manage source selection algorithms
+---
+
+Inventory Management uses the Source Selection Algorithm (SSA) to track the salable quantities of product inventory across all sources and make recommendations for partial and full shipments. 
+
+The SSA uses stocks and sources to check the sales channel for incoming product requests and determines the available inventory. It
```

Comment:
Is "It" meant to be there? I know it means the SSA does the following, hmm maybe: The algorithm: